resources/dicts/events/death/death_reactions/child/child_jealousy.json

```diff
@@ -4,7 +4,9 @@
       "It's sad, the vigil. More so because r_c knows {PRONOUN/r_c/subject} should care and doesn't, than any grief for m_c.",
       "r_c refuses to sit vigil for m_c. {PRONOUN/m_c/subject/CAP} failed r_c as a parent, guardian, and mentor. Why should r_c honor {PRONOUN/m_c/object} now?",
       "r_c is called to m_c's vigil, but declines to speak. {PRONOUN/r_c/subject/CAP}{VERB/r_c/'re/'s} not sure {PRONOUN/r_c/subject}'d have anything good to say.",
-      "They may have shared blood, but nothing more. r_c doesn't show up to m_c's vigil."
+      "They may have shared blood, but nothing more. r_c doesn't show up to m_c's vigil.",
+      "r_c glares at m_c, {PRONOUN/r_c/object} parent's body, with sickingly disgust. Hopefully, {PRONOUN/r_c/subject} could live {PRONOUN/r_c/object} life instead of being a shadow.",
```

Comment:
"sickingly" should be "sickening". also, "glares at [name], their parent's body" doesn't make much sense? i'm not sure how best to explain why it's odd, but it's naming the dead body rather than the person.  it would be better to just omit the name and do "r_c glares at their parent's body"

Reply:
Sorry for the late response, I totally didn't forget again to reply and check with this! ^^; But I've made some changes and removed the following due to it having a similar prompt to those above it:
"Even though the sight of m_c body made a tongue stick out sickly, r_c, without anyone noticing, shed a tear before padding elsewhere."